### PR TITLE
[ADD] cron to compute total overtime

### PR DESCRIPTION
**Pull request by victor-champonnois**
## Odoo task (if applicable)
https://gestion.coopiteasy.be/mail/view?model=project.task&res_id=9216

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: hr_timesheet_overtime/models/hr_employee.py [hunk 1]
```diff
@@ -77,11 +77,11 @@ def _compute_current_day_working_time(self):
...
-    @api.depends(
-        "initial_overtime",
-        "overtime_start_date",
-        "timesheet_sheet_ids.timesheet_overtime_trimmed",
-    )
+    # @api.depends(
+    #     "initial_overtime",
+    #     "overtime_start_date",
+    #     "timesheet_sheet_ids.timesheet_overtime_trimmed",
+    # )
```
Comment (carmenbianca) [inline hr_timesheet_overtime/models/hr_employee.py]: @huguesdk we spent some time on getting this right, but apparently there's a problem still.

Are we sure the reported bug is not fixed by https://github.com/coopiteasy/addons/pull/259 ? This field should _already_ be recomputed daily.

Maybe add an explicit dependency on `company_id.today` or some such?

Comment (victor-champonnois) [reply hr_timesheet_overtime/models/hr_employee.py]: @carmenbianca oups I added the comment for testing, I should remove it !

Comment (carmenbianca) [reply hr_timesheet_overtime/models/hr_employee.py]: It should just work without the cron, no?

I'm not well-read on the real problem.

Comment (victor-champonnois) [reply hr_timesheet_overtime/models/hr_employee.py]: Explanation from cath : En fait le problème vient du fait que le calcul ne se fait que si une modification se fait sur les timesheet donc le cron sert, en plus du calcul, à toutes les nuits forcer le calcul.

Maybe depending on company_id.today could also fix the problem.

Comment (carmenbianca) [reply hr_timesheet_overtime/models/hr_employee.py]: Let's get @huguesdk 's input tomorrow.

**Comment by victor-champonnois** [comment]
Closed because pbm already taken care of by https://github.com/coopiteasy/addons/pull/259

**Pull request closed without merging**